warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexFilterIteratorJexl.java

```diff
@@ -115,7 +115,7 @@ public String toString() {
         } else {
             String upperString = upperBound.toString();
             this.boundingFiRangeStringBuilder.append(upperString.substring(0, upperString.length() - 1));
-            this.boundingFiRangeStringBuilder.append(upperString.charAt(upperString.length() - 1) - 1);
+            this.boundingFiRangeStringBuilder.append((char) (upperString.charAt(upperString.length() - 1) - 1));
```

Comment:
Should line 117 also have a cast to a char?

Reply:
line 117 is appending a string, not a char